<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Simulador</class>
 <widget class="QMainWindow" name="Simulador">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>598</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>965</width>
    <height>598</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayoutMenu">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>230</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayoutContenido" stretch="0">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>965</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbrir">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuSistema">
    <property name="title">
     <string>Sistema</string>
    </property>
    <addaction name="actionSistema_de_Adquisici_on_de_Datos"/>
    <addaction name="actionAsignar_Prueba"/>
    <addaction name="actionAsistente"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionManual"/>
    <addaction name="actionSoporte"/>
   </widget>
   <addaction name="menuAbrir"/>
   <addaction name="menuSistema"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSistema_de_Adquisici_on_de_Datos"/>
   <addaction name="separator"/>
   <addaction name="actionCargar_Prueba"/>
   <addaction name="actionAsignar_Prueba"/>
   <addaction name="actionAsistente"/>
   <addaction name="separator"/>
   <addaction name="actionConsola"/>
   <addaction name="actionControlShmidt"/>
  </widget>
  <action name="actionAsignar_Prueba">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/iconsubir.jpg</normaloff>:/images/iconsubir.jpg</iconset>
   </property>
   <property name="text">
    <string>Asignar Prueba</string>
   </property>
   <property name="toolTip">
    <string>Asignar Prueba</string>
   </property>
  </action>
  <action name="actionCargar_Prueba">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/cargarPrueba.jpeg</normaloff>:/images/cargarPrueba.jpeg</iconset>
   </property>
   <property name="text">
    <string>Cargar Prueba</string>
   </property>
   <property name="toolTip">
    <string>Cargar Prueba</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/iconSalir.jpg</normaloff>:/images/iconSalir.jpg</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
  <action name="actionSoporte">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/iconAcerca.jpg</normaloff>:/images/iconAcerca.jpg</iconset>
   </property>
   <property name="text">
    <string>Acerca</string>
   </property>
  </action>
  <action name="actionSistema_de_Adquisici_on_de_Datos">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/conexion.gif</normaloff>:/images/conexion.gif</iconset>
   </property>
   <property name="text">
    <string>Sistema de Adquisici√≥n de Datos</string>
   </property>
   <property name="shortcut">
    <string>CapsLock, CapsLock, CapsLock, CapsLock</string>
   </property>
  </action>
  <action name="actionConsola">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/consola.png</normaloff>:/images/consola.png</iconset>
   </property>
   <property name="text">
    <string>Consola</string>
   </property>
  </action>
  <action name="actionControlShmidt">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/cidaIcon.png</normaloff>:/images/cidaIcon.png</iconset>
   </property>
   <property name="text">
    <string>ControlShmidt</string>
   </property>
   <property name="toolTip">
    <string>ControlShmidt</string>
   </property>
  </action>
  <action name="actionAsistente">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/ico_soporte.png</normaloff>:/images/ico_soporte.png</iconset>
   </property>
   <property name="text">
    <string>Asistente</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
